.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,11 +3,19 @@ Thank you for submitting a PR to pylint!
 
 To ease the process of reviewing your PR, do make sure to complete the following boxes.
 
-- [ ] Write a good description on what the PR does.
-- [ ] Create a news fragment with `towncrier create <IssueNumber>.<type>` which will be
-  included in the changelog. `<type>` can be one of: breaking, user_action, feature,
-  new_check, removed_check, extension, false_positive, false_negative, bugfix, other, internal.
-  If necessary you can write details or offer examples on how the new change is supposed to work.
+- [ ] Document your change, if it is a non-trivial one.
+  - A maintainer might label the issue ``skip-news`` if the change does not need to be in the changelog.
+  - Otherwise, create a news fragment with ``towncrier create <IssueNumber>.<type>`` which will be
+    included in the changelog. ``<type>`` can be one of: breaking, user_action, feature,
```

Comment:
Let's just link to the file that defines this.

Reply:
Can we include a file in the github template ? Or should contributor click a link ? Not sure they're going to do it if they have to click :D

Reply:
I don't think so, but just a file path should work I think!

Reply:
I'm not sure I understand what you mean, could you suggest the edit ?